config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

```diff
@@ -283,8 +283,6 @@ periodics:
       - --gcp-master-size=n1-standard-2
       - --gcp-node-size=n1-standard-4
       - --gcp-nodes=3
-      - --gcp-project=k8s-jenkins-gci-kubemark
-      - --gcp-zone=us-central1-f
```

Comment:
you probably still need the --gcp-zone flag

Reply:
🤦‍♂️ yeah, we should probably give this a default value..

Reply:
Could we change the default to `us-central1-f` in kubetest? (as that seems to be used by most of jobs)

Reply:
SGTM

Reply:
ideally we should probably pick a random zone? but I guess `us-central1-f` can be a good choice to start with.

also looks like you cannot have --zone and --region set at the same time? maybe we should unify it as a --location flag?